package.json

```diff
@@ -21,6 +23,9 @@
     },
     "devDependencies": {
         "@types/node": "^12.0.4",
-        "typescript": "^3.5.2"
+        "typescript": "3.9.2",
+        "jest": "^26.0.1",
+        "@types/jest": "^25.2.2",
+        "ts-jest": "^25.5.1"

```

Comment:
What does `ts-jest` do?

Reply:
import { mocked } from 'ts-jest/utils'. Used for mocking.